internal/constants/constants.go

```diff
@@ -24,8 +24,9 @@ const (
 	AWSSecretAccessKey           = "aws_secret_access_key" //nolint:gosec
 	AWSAccessKeyID               = "aws_access_key_id"
 	AWSRoleSessionName           = "cluster-logging" // identifier for role logging session
-	AWSCredentialsKey            = "credentials"     // credrequest key to check for sts-formatted secret
-	AWSWebIdentityRoleKey        = "role_arn"        // manual key to check for sts-formatted secret
+	AWSCredentialsConfigMapName  = "aws-creds-configmap"
```

Comment:
We are creating a "configmap".  We can drop "configmap" from the name.  It will help with my concerns about the name exceeding the name limits